runtime/vm/classsupport.c

```diff
@@ -610,13 +610,13 @@ callFindLocallyDefinedClass(J9VMThread* vmThread, J9Module *j9module, U_8* class
 
 		findResult = (IDATA) returnVal;
 		if (0 == findResult) {
-			TRIGGER_J9HOOK_VM_FIND_LOCALLY_DEFINED_CLASS_FROM_FILESYSTEM(vmThread->javaVM->hookInterface, 
-																		 vmThread, 
-																		 (const char *)className, 
-																		 classNameLength,
-																		 classLoader,
-																		 &classFound,
-																		 returnPointer);
+			TRIGGER_J9HOOK_VM_FIND_LOCALLY_DEFINED_CLASS_FROM_FILESYSTEM(vmThread->javaVM->hookInterface,
+					vmThread,
```

Comment:
missed to put the first input parameter on a new line
```suggestion
			TRIGGER_J9HOOK_VM_FIND_LOCALLY_DEFINED_CLASS_FROM_FILESYSTEM(
					vmThread->javaVM->hookInterface,
					vmThread,
```